libs/core/src/lib/combobox/combobox.component.html

```diff
@@ -15,7 +15,9 @@
         <fd-popover-control>
             <ng-container *ngTemplateOutlet="controlTemplate"></ng-container>
         </fd-popover-control>
-        <fd-popover-body [hidden]="!displayedValues || !displayedValues.length">
+        <fd-popover-body
+            [hidden]="!displayedValues || !displayedValues.length"
+            [class.fd-popover__popper--no-border-radius]="true">
```

Comment:
Could we use it like that?
```suggestion
            class="fd-popover__popper--no-border-radius">
```

Reply:
Done